Resources/Prototypes/_DV/Body/Organs/Animal/slimes.yml

```diff
@@ -0,0 +1,18 @@
+- type: entity
+  id: SentientSmileCore
+  parent: SentientSlimesCore
+  name: sentient smile core
+  suffix: Do Not Map
```

Comment:
organs are fine to map

Reply:
There shouldn't be more than 1 of these in a map though. And the spawner is handling that.

Reply:
It doesn't ensure only 1 but it's less likely a new mapper will decide to add an additional one with a suffix.

You're right that it's not needed. I could tag->hide menu the spawner. But I was trying to save mappers an extra step for replacing it on current maps. If you prefer we just hide the spawner and map the brain, that's fine with me too. Just an extra step.

Reply:
I guess I could also migrate the spawner ->brain. Let me know which route you prefer.

Reply:
yeah just leave the spawner alone + migrate it to the brain item and map the brain instead, makes map loading 0.00001% faster :trollface: